### DISCUSSION TITLE
我的表达欲在减少

### DISCUSSION THREAD

**Discussion by giscus** [zu1k/blog#47] [Comments]
# 我的表达欲在减少

翻看近几年我发过的朋友圈，19年1条，20年2条，21年发了3条，而今年仅有一条新年祝福。发朋友圈和说说频率的锐减，我在很早之前就发现了，博客更新频率也在减少，与人倾诉的频率几乎归零。不知道为什么，随着知识的增加、人生经验的积累，我的表达欲却在慢慢的下降，而现在几乎已经到达一个极低的水平。

https://zu1k.com/posts/thinking/low-desire-of-expression/#giscus

**Comment by dp0d** [comment]
你只是长大了，老弟

**Comment by Alafun** [comment]
我也是,但我很菜

**Comment by Evanavevan** [comment]
越长大越孤单 你不是一个人

**Comment by Maxlinn** [comment]
同，初生牛犊的时候受鼓动，运营了自己的博客。现在看来，博文内容相当粗糙，但当时乐在其中，也获得了一些不大不小的关注。

反而读了研之后，很少再发布博文了，大部分内容记在知识库里，知识库也是不公开的。

我希望我的内容，既是深入的又是浅显的，既是严肃的又是有趣的，能够满足所有人的需求，让所有人都挑不出刺来。

结果就是下笔万钧重，常常一段话写完，删除，换一种更准确/更直观表述。或者写到某个句子，觉得自己要印证一下，不然是对读者不负责。最后长叹一声麻烦，贴到知识库里，不写了。

现在算是否定之否定了。不再期望满足所有人，写作时完全按服务自己来写。同时承认自己的局限性，不再追求滴水不漏，尽力而为就好。现在又找回了当初写博客的快乐：记一些事，帮一些人。

**Reply by zu1k** [reply]
透彻！就是这个感觉，一模一样

**Comment by howie6879** [comment]
我是从你的deepl api文章过来的，但是看到这篇文章想说点什么。

我个人认为你是成长了，但可能有些事情还在思考，现在大可不必就下结论。可能在后续读书、阅历增长后会想明白；人都会有这样一个过程，自信且喜欢思考或者说喜欢专研的人更容易陷进去，会陷入到孤独迷茫的情绪里面去。这个过程是美好的，也是痛苦的，很多人经历过，我也是。

而且你的表达欲应该没有减少，开源项目、博客、任何你创作的东西都可以是你表达欲的实际体现，看你描述更偏向于匿名平台发表，不知道你有没有使用flomo。

我觉得你只是更加谨慎了，是你内心认为自己是个优秀的人，你有自己的原则与底线，而这个底线你不太希望让人看到。你是不是私下会非常努力产出然后也乐意将自己努力的成果贡献给别人以提供帮助的人，你是具有一些领导力的人，也是一个思考者与产出者，不知你是否认同。

结合你说的，我有个建议：

> 以自己为主吧，以自己进步为第一目标，只要产出的东西是在自己认知范围边界努力而成的产物，都可大方输出，考虑太多是累赘，输出是一种进步，这不必解释。其实读者也可以看做让你进步的催化剂，他们的意见，想法都可以让你反向吸收。

输出是一个创造的过程，你可以看看《张艺谋的作业》。72岁的老爷子了，功成名就还在创造，他任何一个作品都在被亿万人民审视，可他只关注自己是否在创作的路上持续进步。

欢迎交流。

**Reply by zu1k** [reply]
请允许我叫你老胡，这种感觉就像遇到知己，我从未想过一个陌生人可以通过我的文章、开源项目，能够如此的了解我、了解我心里的想法，我希望从此刻起我们不再是陌生人。

自信，又害怕自负，这应该就是我目前的状态。我常常会给自己一些标准，但有时候却发现自己无法完成，心里很难受，不想潦潦草草完成，最终结果往往是中途放弃。也许是过分自信，其实就是自负，回过头来看看，很多时候自己定下的标准确实有点高，自己给自己心理上背上了包袱，到头来限制了发挥。

我确实喜欢思考，其实就是胡思乱想，想各种大大小小的事，想这个社会这个世界的运作机理，想未来，想人性。观察的越多，思考的越多，慢慢的发现新的想法与之前的想法出现了背离，这个时候却无人给我解答。我不曾向人问过，我觉得向身边人倾诉和咨询这些想法非常难为情，有时候我自负的认为他们都不懂，我不认为他们经历过，我不认为他们思考过这些问题。我开始慢慢向书中去汲取，开始看一些心理学、哲学相关的内容，即使晦涩即使难懂。

我确实开始倾向于匿名，你知道这是非常矛盾的心理，想让人看到自己的亮点、想让人看到自己获得的成果，又不想让人了解我这个人，就好像要把自己藏起来，然后在江湖上留一个无人不知的名号，但是谁也没见过这个人。

对于你给的建议，非常感谢！我觉得自己开始着手改变了，并不是很晚，我还年轻。

**Reply by howie6879** [reply]
我们当然自此不再是陌生人，互联网的乐趣就在于此不是吗？

**对于标准**：

达利欧的《原则》这本书说了工作生活中的一些原则，如你思考了你需要什么，但还要结合现实，知道了现实后你才可以确定如何行动。

你有自己的标准，这是好事，但要考虑这个标准也可以说是目标是否远远超出了所在的舒适区，目标最好在舒适区外面一点然后努力能达到的，所以目标需要合理且现实，不然造成的反效果还是很大的。

**对于思考**：

我想对你说声恭喜，不知道你是否看过这张图：

![image](https://user-images.githubusercontent.com/17047388/162445352-85562722-1ea7-487c-b95c-c43d317a95c1.png)

你再也不在愚昧之巅了！你肯定过了刚学程序就要在朋友圈展示你的hello world，你应该在谷底正慢慢往上爬，这个过程需要不断读书、思考、实践来提升，考虑到你还在读书，社会阅历需要真正的工作来磨炼，不要急，先读书，慢慢你就上来了，然后会感觉非常美妙，你会轻松地控制自己的情绪，对于一些事情你会有自己的理解，看到一个信息会有自己的判断。

但注意，这个过程是循环的，很可能自以为开悟了实际上是一个新的愚昧之巅，这很有意思，对吧。这是一个不断往上爬，解决了部分信息不对称又不断思考往上爬的过程。

**对于输出**：

一定要坚持输出，匿名的和博客全都要，匿名的反省自己，自我思考，记录想法，对外博客展示自己创造的一切，你一定要是创造者而不是实施者。

**最后**：

我不过虚长你几岁，就像你觉得我了解你一样，你也肯定了解我，我和你是同类人，你经历过的状态我经历过，我现在的状态你不久也会经历。

包括现在，我能顺心畅谈的朋友也不多，但我找到了更好的寄托方式，我现在致力于提升生活中的幸福感，我所做的一切事情，包括工作、学习都是围绕这块，人真的很有意思，每人脑中都有一个世界，宏大又精妙。

祝好~欢迎交流。

**Comment by uoox** [comment]
我也是这样，不过……尽管写吧，可能你觉得写的那些东西有你说的各种问题，但对我帮助很大，我有学到东西，很有用，谢谢！

**Comment by crazypeace** [comment]
推荐你看看这个
https://m.youtube.com/watch?v=bESRxEFuqP4&t=4s
“你不需要是个天才，才有资格分享内容”

**Comment by yuantuo666** [comment]
博主加油，从ChatGPT找过来的。
看到你折腾校园网的文章，想起自己和你是一样的方法，但感到惋惜的是当时没有找到你的这篇文章。
你的很多文章都很有意思，不是所有人刚接触这个领域就是大神（我是个菜鸡），所以像您这样大佬写出来的文章我都是细细品读。
看到博主更新越来越不频繁了，希望博主继续加油，不论好与坏，就当做是记录的一种方式呗~

**Comment by Aikoin** [comment]
同intp，夕阳那段描写莫名很感动。。以及似乎是同级生，很多想法也真的好像，大半夜激动的发了几个想法来记录。。我也是表达欲越来越弱，但是，得表达啊！得记录啊！我在试着不去害怕那些牵绊着我的东西了

**Comment by Evanavevan** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，你发的邮件我已经收到，我会尽快阅读你的邮件的

**Comment by shyu216** [comment]
加油，陌生人！

**Comment by Evanavevan** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，你发的邮件我已经收到，我会尽快阅读你的邮件的
